scripts/madrid/predictions_collapsed/logistic_regression/Snakefile

```diff
@@ -0,0 +1,61 @@
+'''
```

Comment:
Please change this Snakefile to match filestructure of latest master branch.

See e.g. 
https://github.com/tufts-ml/time_series_prediction/tree/master/scripts/unimib_shar_activities/predictions_collapsed

Reply:
Most of the front env variable setting can be moved to a `config_loader.py` script

Reply:
Changed file-structure to match : 
https://github.com/tufts-ml/time_series_prediction/tree/master/scripts/unimib_shar_activities/predictions_collapsed

Reply:
Do we want all the data-prep scripts from madrid-data-prep in the time-series predict pipeline? I'm a bit confused about how to edit the config_loader.py. For now the front env variables are in the smk files.